src/test/java/com/aws/greengrass/shadowmanager/sync/strategy/RealTimeSyncStrategyTest.java

```diff
@@ -137,17 +139,16 @@ void GIVEN_sync_request_WHEN_queue_throws_interrupted_exception_THEN_sync_reques
     }
 
     @Test
-    void GIVEN_request_queue_WHEN_put_and_clear_THEN_queue_has_correct_number_of_requests() throws InterruptedException {
+    void GIVEN_request_queue_WHEN_put_and_clear_THEN_queue_has_correct_number_of_requests() {
         strategy = new RealTimeSyncStrategy(executorService, mockRetryer);
 
         strategy.syncing.set(true);
 
-        Random rand = new Random();
-        int randomNumberOfSyncRequests = rand.nextInt(1024);
-        for (int i = 0; i < randomNumberOfSyncRequests; i++) {
+        int numberOfSyncRequests = 100;
```

Comment:
Curious about this change? is the random number causing issue?

Reply:
This was something I was trying. The idea was to log less cause the test logs were blowing up. Going north of 140 MB every time. 

To be honest, this test does not need to be random. This just checks that if we clear a request queue after having some requests already in it, it removes all of them.

Reply:
are we logging too much stuff in general then? if we have 140MB of logs - should we consider bumping stuff down a log level?

Reply:
Not sure why it was logging that much. It was just logging that we were executing a sync request. We can turn that down to DEBUG. but I don't think it should be the case in real time.